fabric-fluid-v1/src/main/java/net/fabricmc/fabric/api/fluids/v1/FluidView.java

```diff
@@ -29,7 +29,7 @@
 import net.fabricmc.fabric.api.fluids.v1.world.SidedFluidContainer;
 
 /**
- * A util class for accessing fluid containers.
+ * An util class for accessing fluid containers.
```

Comment:
Original was grammatically correct

Reply:
no

Reply:
English is a cursed language, turns out you're right. My bad.

Reply:
wait really? but there's a vowel right there

Reply:
my 4th grade english teacher would like to have a word with you

Reply:
![image](https://i.imgur.com/f0PiyZV.png)

English is cursed.

Reply:
what in the world